include/entity.h

```diff
@@ -2039,3 +2028,5 @@ SYNC_FIELD(ET_Subweapon, ET_Agunea, subweaponId);
 SYNC_FIELD(ET_Subweapon, ET_AguneaCrash, subweaponId);
 SYNC_FIELD(ET_Subweapon, ET_GiantSpinningCross, subweaponId);
 SYNC_FIELD(ET_Subweapon, ET_CrashCross, subweaponId);
+
+SYNC_FIELD(ET_Generic, ET_Player, unkB8)
```

Comment:
Rather than aligning across all ETs, this is why I think we should instead change this to be part of Entity, and not part of Ext.

Reply:
The problem still exists if it comes after `ext` since I'm not sure all the `ext`s have the same size

Reply:
But Ext is a union, so it will take the size of its largest member. Right now Ext is 0x64 bytes. In my imagined future, we would remove unkB8, so Ext would be 0x60 bytes, and then we would have the 4-byte unkB8 right after.